.github/workflows/builds-review.yaml

```diff
@@ -0,0 +1,71 @@
+name: Review builds
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  build:
+    if: |
+      github.event_name == 'pull_request' &&
+      github.event.label.name == 'build::review'
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-latest
+            subdir: linux-64
+          - runner: macos-latest
+            subdir: osx-64
+          - runner: windows-latest
+            subdir: win-64
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - name: Check current actor
+        id: in_team
+        uses: conda/actions/user-in-team
+        with:
+          org: conda
+          team: conda-core
+          user: ${{ github.actor }}
```

Comment:
@kenodegard Does that look right?

Reply:
Yup - just need to add the new release ref
```yml
uses: conda/actions/user-in-team@v22.1.2
```

Reply:
Super good, update comming..